### PR TITLE
Implement RequestUpdateAudioRendererAuto with a method

### PR DESCRIPTION
**Pull request by greggameplayer**

### CODE REVIEW

File: Ryujinx.HLE/OsHle/Services/Aud/IAudioRenderer.cs [hunk 1]
```diff
@@ -35,13 +36,18 @@ public IAudioRenderer(AudioRendererParameter Params)
...
         {
-            long OutputPosition = Context.Request.ReceiveBuff[0].Position;
-            long OutputSize     = Context.Request.ReceiveBuff[0].Size;
-
+            return RequestUpdateAudioRendererMethod(Context, Context.Request.ReceiveBuff[0].Position, Context.Request.ReceiveBuff[0].Size, Context.Request.SendBuff[0].Position);
+        }
+        
+        public long RequestUpdateAudioRendererAuto(ServiceCtx Context)
+        {
+            return RequestUpdateAudioRendererMethod(Context, Context.Request.GetBufferType0x22().Position, Context.Request.GetBufferType0x22().Size, Context.Request.GetBufferType0x21().Position);
```
Comment (gdkchan) [inline Ryujinx.HLE/OsHle/Services/Aud/IAudioRenderer.cs]: Please don't create those long lines, they are hard to read. Put each argument on a separate line. See what I did on the recent audout PR.

File: Ryujinx.HLE/OsHle/Services/Aud/IAudioRenderer.cs [hunk 2]
```diff
@@ -35,13 +36,18 @@ public IAudioRenderer(AudioRendererParameter Params)
...
+        }
+        
+        public long RequestUpdateAudioRendererMethod(ServiceCtx Context, long OutputPosition, long OutputSize, long InputPosition)
```
Comment (gdkchan) [inline Ryujinx.HLE/OsHle/Services/Aud/IAudioRenderer.cs]: Same thing: arguments on new lines.

File: Ryujinx.HLE/OsHle/Services/Aud/IAudioRenderer.cs [hunk 3]
```diff
@@ -93,7 +99,7 @@ public long RequestUpdateAudioRenderer(ServiceCtx Context)
 
             //TODO: We shouldn't be signaling this here.
             UpdateEvent.WaitEvent.Set();
-
+
```
Comment (gdkchan) [inline Ryujinx.HLE/OsHle/Services/Aud/IAudioRenderer.cs]: Unnecessary spacing on empty line.

File: Ryujinx.HLE/OsHle/Services/Aud/IAudioRenderer.cs [hunk 4]
```diff
@@ -35,13 +36,18 @@ public IAudioRenderer(AudioRendererParameter Params)
 
         public long RequestUpdateAudioRenderer(ServiceCtx Context)
         {
-            long OutputPosition = Context.Request.ReceiveBuff[0].Position;
-            long OutputSize     = Context.Request.ReceiveBuff[0].Size;
-
+            return RequestUpdateAudioRendererMethod(Context, Context.Request.ReceiveBuff[0].Position, Context.Request.ReceiveBuff[0].Size, Context.Request.SendBuff[0].Position);
```
Comment (gdkchan) [inline Ryujinx.HLE/OsHle/Services/Aud/IAudioRenderer.cs]: Please don't create those long lines, they are hard to read. Put each argument on a separate line. See what I did on the recent audout PR.

File: Ryujinx.HLE/OsHle/Services/Aud/IAudioRenderer.cs (hunk 2 above)
Comment (gdkchan) [inline Ryujinx.HLE/OsHle/Services/Aud/IAudioRenderer.cs]: `RequestUpdateAudioRendererMethod` -> `RequestUpdateAudioRendererImpl` sounds better IMO.

**Comment by greggameplayer** [comment]
Can't modifie it i have delete my repo and refork this repo

**Comment by gdkchan** [comment]
Ok, then open a new PR with the fixes if possible.

**Comment by greggameplayer** [comment]
ok i will done that

**Pull request closed without merging**